libnavigation-router/src/test/resources/multi_leg_route_refreshed_second_leg.json

```diff
@@ -38,7 +38,7 @@
       "weight_typical": 1511.56,
       "routeIndex": "0",
       "distance": 2839.016,
-      "duration": 623.8819999999998,
+      "duration": 845.8819999999998,
```

Comment:
The refresh response in this test case swaps the data in a waypoint to be a charger with charge time `222`.